modules/installation-vsphere-machines.adoc

```diff
@@ -62,30 +62,29 @@ endif::openshift-origin[]
 .. Click the *VMs and Templates* view.
 .. Right-click the name of your datacenter.
 .. Click *New Folder* -> *New VM and Template Folder*.
-.. In the window that is displayed, enter the folder name. If you did not specify an existing folder in the `install-config.yaml` file, then create a folder with the same name as the infrastructure ID.
+.. In the window that is displayed, enter the folder name. If you did not specify an existing folder in the `install-config.yaml` file, create a folder with the same name as the infrastructure ID.
 
-. In the vSphere Client, create a template for the OVA image.
+. In the vSphere Client, create a template for the OVA image and then clone the template as needed.
 +
 [NOTE]
 ====
-In the following steps, you use the same template for all of your cluster machines and provide the location for the Ignition config file for that machine type when you provision the VMs.
+In the following steps, you create a template and then clone the template for all of your cluster machines. You then provide the location for the Ignition config file for that cloned machine type when you provision the VMs.
```

Comment:
Maybe my ignorance in action. Are _clone the template for all of your cluster machines_ and _cloned machine type_ the same thing?

Reply:
It is, yes. The machine type being `compute` or `control plane`, for example. I agree that the wording is a bit odd but going to leave "machine type" because we refer to "machine type" in several other places in this procedure. And since it was there previously, I don't want to inadvertently change the meaning. :)